app/Resources/views/committee/timeline/_message.html.twig

```diff
@@ -1,5 +1,5 @@
 {% if is_granted('ROLE_ADHERENT') %}
-    <div class="text--body">{{ item.content|raw }}</div>
+    <div class="committee-event-description text--body">{{ item.content|raw }}</div>
```

Comment:
@ngrilly I think neither of those would work for the event page. Currently on master, there's only one `committee-event-description`, and it's on the event page. Which makes sense. But having `feed` on that page wouldn't work. 

What I would suggest is adding a new utility class. Something like `text--has-list` or `text--with-list`, or something like that. The same way we have `text--breakword` or `icon--with-text`, in the `text.scss` file. You could also make it a mixin if and import it where you need it, and make a class of it where you don't have a class you can put it under. 

Yes, and @tgalopin is right about BEM. I think somehow the timeline and a few other admin pages don't follow this convention but where we're using BEM, we should use BEM. 

Let me know if that makes sense! And sorry, I hadn't seen those responses until now. Feel free to ping me on Slack in the future if I'm slow!